components/layout/OpenAuthoring.tsx

```diff
@@ -31,9 +31,10 @@ export const OpenAuthoring = ({ children }) => {
 
   const updateAuthChecks = async () => {
     setGithubAuthenticated(await isGithubTokenValid())
-
-    const forkName = Cookies.get('fork_full_name')
-    setForkValid(await isForkValid(forkName))
+    const forkFullName = Cookies.get('fork_full_name')
+    if (forkFullName) {
```

Comment:
if we don't have a fork_full_name cookie, we won't call `setForkValid` (but it should set it to be invalid). I think we will want to remove this check since isForkValid will return false when the cookie value doesn't exist